index.html

```diff
@@ -1,110 +1,16 @@
 ---
 layout: default
 ---   
-<div class="greenhead">Where can I see the RFP?</div>
+
+{% for faq in site.data.faq %}
```

Comment:
Here's an example of an implementation of the page-specific layout:

+ [page layout](https://github.com/inaki/cities/blob/master/_layouts/page.html)
+ [includes file](https://github.com/inaki/cities/blob/master/_includes/fellows.html)
+ [individual page example](https://github.com/inaki/cities/blob/master/pittsburgh/index.md) ([view raw](https://raw.githubusercontent.com/inaki/cities/master/pittsburgh/index.md) for a better sense of what's going on)